### PR TITLE
Addition of Azure documentation

### PR DESCRIPTION
**Pull request by jeniawhite**
Added Azure documentation in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,86 @@ NOTE: at present, only some components have transitioned to this new model, the
...
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    namespace: kube-system
+    name: openstack-credentials
+  data:
+    clouds.yaml: Base64encodeCloudCreds
+    clouds.conf: Base64encodeCloudCredsINI
+  ```
+  5. VSphere:
```
Comment (joelddiaz) [inline README.md:81]: so close to alphabetical. can we move VSphere to the bottom of the list?

Comment (jeniawhite) [reply README.md]: @joelddiaz 
Done.

**Comment by joelddiaz** [comment]
@jeniawhite one last thing, can you squash this down into a single commit? thanks for the PR!

**Comment by jeniawhite** [comment]
@joelddiaz 
Squashed this down into a single commit.
Thank you.

**Pull request merged**